library/packages/src/modules/SlideShow.rb

```diff
@@ -1029,6 +1042,7 @@ def GetSetup
     publish variable: :language, type: "string"
     publish variable: :widgets_created, type: "boolean"
     publish variable: :user_switched_to_details, type: "boolean"
+    publish variable: :user_switched_to_release_notes, type: "boolean"
```

Comment:
Is it needed to have those variable publicly published? It looks to me like internal variable.

Reply:
ah, I just found it is used in PackageSlideShow where is another logic when you change CD. :man_facepalming: